src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp

```diff
@@ -303,6 +308,7 @@ namespace OpenMS
     found_adduct_(source.found_adduct_),
     empirical_formula_(source.empirical_formula_),
     matching_hmdb_ids_(source.matching_hmdb_ids_),
+    matching_names_(source.matching_names_),
```

Comment:
can't those constructors be defaulted?

Reply:
yes I think so, did not check thoroughly yet.